1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,30 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+
+function checkGroupMember(group,students)) {
+  // let isStudents = true;
+   for (let i = 0; i < group.length; i++) {
+    for (let i = 0; j < students.length; i++) {
+      if (group[i] === students[i]) {
+      
+      }
+    }
+    return false;
+  }
+  return true;
+}
+
+let groupIsOnlyStudents = checkGroupMembers(group, studentss);
+
+if (groupIsOnlyStudents) {
+  console.log("The group contains only students");
+} else {
+  console.log("The group does not contain only students");
+}
+  
+
+let groupIsOnlyStudents = GroupMembers(group,students); // complete this statement
```

Comment:
This line and below are duplicates. The code as a whole looks good at a first glance but it does not compile. There are quite a few bits to work through here, you need to run the code, see where it falls over, and fix the problem - repeat until it's fixed